games/tss/base_files/config.json

```diff
@@ -0,0 +1,55 @@
+
+{  
+    "name": "Twinkle Star Sprites",
+    "smashgg_game_id": 40611,
+    "challonge_game_id": 1843,
+    "character_to_codename": {
+    "Arthur": {
+      "codename": "arthur"
+    },
+    "Dark Ran": {
+      "codename": "darkran"
```

Comment:
Dark Ran's icon doesn't display, I think the icon file name isn't using the correct codename

Reply:
I fixed the filename in my fork. Will that fix it here?

Reply:
Yep, it's fixed on my end, I'll merge this so it can be available in the asset downloader. Thanks!